sdk/cosmos/azure-cosmos/azure/cosmos/aio/_database.py

```diff
@@ -202,6 +203,9 @@ async def create_container(
         :keyword int analytical_storage_ttl: Analytical store time to live (TTL) for items in the container.  A value of
             None leaves analytical storage off and a value of -1 turns analytical storage on with no TTL. Please
             note that analytical storage can only be enabled on Synapse Link enabled accounts.
+        :keyword Dict[str, Any] vector_embedding_policy: The vector embedding policy for the container. Each vector
```

Comment:
Can we be explicit about documenting the preview/provisional nature of this feature in all the docstrings?

Reply:
Looks like this is only missing in a couple of places.